app/vmctl/README.md

```diff
@@ -432,6 +436,79 @@ Found 2 blocks to import. Continue? [Y/n] y
 2020/02/23 15:51:07 Total time: 7.153158218s
 ```
 
+## Migrating data by remote read protocol
+
+`vmctl` supports the `remote-read` mode for migrating data from any database which supports Prometheus remote read protocol 
+to VictoriaMetrics time-series database.
+
+See `./vmctl remote-read --help` for details and full list of flags.
+
+Migration is based on the remote reading and allows transparently fetch samples. 
+Remote read protocol supports two different modes `SAMPLES` and `STREAMED_XOR_CHUNKS`. You can read more about it 
+[here](https://prometheus.io/blog/2019/10/10/remote-read-meets-streaming/).
+
+Please notice at that moment not all TSDB support `STREAMED_XOR_CHUNKS` mode and can handle stream requests.
+
+To start the migration process some flags should be defined:
+1. `--remote-read-src-addr` - address to perform read from;
+2. `--vm-addr` - address for single-node VM is usually equal to `--httpListenAddr`, and for cluster version;
+is equal to `--httpListenAddr` flag of vminsert component (for example `http://<vminsert>:8480/insert/<accountID>/prometheus`);
+3. `--remote-read-filter-time-start` - the time filter in RFC3339 format to select timeseries with timestamp equal or higher than provided value. E.g. '2020-01-01T20:07:00Z';
+4. `--remote-read-step-interval` - split export data into chunks. Valid values are `month, day, hour, minute`;
+
+As soon as required flags are provided and all endpoints are accessible, `vmctl` will start the fetch process 
+from remote read API. Basically, it just fetches all timeseries from the provided database and sends import requests 
+for each timeseries and pass results to VM importer. VM importer then accumulates received samples in batches and 
+sends import requests to VM.
+
+The importing process example for local installation of Prometheus
+and single-node VictoriaMetrics(`http://localhost:8428`):
+
+```
+./vmctl remote-read \
+--remote-read-src-addr=http://127.0.0.1:9091 \
+--remote-read-filter-time-start=2021-10-18T00:00:00Z \
+--remote-read-step-interval=hour \
+--vm-addr=http://127.0.0.1:8428 \
+--vm-concurrency=6
```

Comment:
why do you need to specify concurrency in example?

Reply:
There was an issue with show statistics at the end of the process. 
When we add this flag part of the output hides the statistic when the process is finished.  
So I added this flag to see that the whole information is accessible to the user.

Reply:
I don't get it( How concurrency affects the output?